src/common/sorted-table/index.js

```diff
@@ -137,6 +147,17 @@ class ColumnHead extends Component {
 const DEFAULT_ITEMS_PER_PAGE = 10
 
 @propTypes({
+  groupedActions: propTypes.arrayOf(propTypes.shape({
+    handler: propTypes.func.isRequired,
+    icon: propTypes.node.isRequired,
+    label: propTypes.node,
+    size: propTypes.string
+  })),
+  individualActions: propTypes.arrayOf(propTypes.shape({
+    handler: propTypes.func.isRequired,
+    icon: propTypes.node.isRequired,
+    btnStyle: propTypes.string
+  })),
```

Comment:
In the individual actions, we need the label prop and in the grouped actions we need the action style prop. That's why i use the two components ActionRow and ActionBar.